R/setup.R

```diff
@@ -416,9 +416,9 @@ AgProductionTechnology_setup <- function(aLandLeaf, aAgData, aScenarioInfo) {
     }
 
     # Set cost in the LandLeaf
-    if(aScenarioInfo$mUseZeroCost == FALSE &&
-       name %in% unique(cost$AgProductionTechnology) && y %in% unique(cost$year))  {
-      aLandLeaf$mCost[per] <- as.numeric(cost$nonLandVariableCost[cost$year == y &&
+    if(aScenarioInfo$mUseZeroCost == FALSE &
+       name %in% unique(cost$AgProductionTechnology) & y %in% unique(cost$year))  {
+      aLandLeaf$mCost[per] <- as.numeric(cost$nonLandVariableCost[cost$year == y &
```

Comment:
FYI, this is the one where I made a mistake.  `cost$year == y` and `cost$AgProductionTechnology == name` are both logical vectors and need to be combined with (single) `&`.